platform/linux-generic/odp_timer.c

```diff
@@ -1254,6 +1254,8 @@ int odp_timer_capability(odp_timer_clk_src_t clk_src,
 	capa->max_tmo.res_hz  = timer_global->highest_res_hz;
 	capa->max_tmo.min_tmo = 0;
 	capa->max_tmo.max_tmo = MAX_TMO_NSEC;
+	capa->sched_queue_enq = true;
+	capa->plain_queue_enq = true;
```

Comment:
API and implementation changes should be in separate patches. So, move this into a "linux-gen: timer: ..." patch.

Reply:
Will split the pact in next verison.
Thanks